plugins/etcd/pom.xml

```diff
@@ -27,7 +27,6 @@
       <dependency>
          <groupId>commons-io</groupId>
          <artifactId>commons-io</artifactId>
-         <version>2.4</version>
```

Comment:
`commons-io` is only required for `etcd` plugin.
Why we should move to `plugins/pom.xml` ?

Reply:
@diegolovison  Actually I didn't move, I only added a `dependencyManagement` just to be able to remove the `version` from `plugins/etcd/pom.xml`. IMO is more organised to avoid add versions spread into the project